test/unit/authorizer_test.rb

```diff
@@ -10,169 +10,201 @@ def setup
     @permission = FactoryGirl.create(:permission, :host)
   end
 
-  # limited, unlimited, permission with resource, without resource...
-  test "#can?(:view_hosts) with unlimited filter" do
-    FactoryGirl.create(:filter, :role => @role, :permissions => [@permission])
-    auth = Authorizer.new(@user)
-
-    assert auth.can?(@permission.name.to_sym)
-    refute auth.can?(:view_domains)
-  end
-
-  test "#can?(:view_hosts) with unlimited filter" do
```

Comment:
It should probably say "with wildcard filter". The previous test has a nil/unlimited search, this has `name = *`.

The duplicate name shouldn't be an issue in minitest with `test`, only when using `def`.